src/main/webapp/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <script src="/scripts/map.js"></script>
+    <script src="/scripts/to-top.js"></script>
```

Comment:
Instead of having a function-specific `js` file, I would recommend having a page-specific `js` file (like `index.js`) so that we don't have to create new `js` files for potential new functions related to this page.

Reply:
Thanks!